examples/meta/src/meta_api/getters.sg

```diff
@@ -0,0 +1,2 @@
+Kernel k = kernel("GaussianKernel")
```

Comment:
could you remove this file? it makes the build fail as the method is not implemented.
I will use things in the feature branch